plugin/taboo.vim

```diff
@@ -172,6 +172,9 @@ fu s:wincountUnicode(tabnr, ubiquitous)
 endfu
 
 fu s:modflag(tabnr)
+    if !g:taboo_tabline
+        return ""
+    endif
```

Comment:
It would be a pity to lose the modification flag. I think it would be better to put this inside the loop  (at line  180) and just return `g:taboo_modified_tab_flag` if `g:taboo_tabline` is false.